samples/helloshell/prj_qemu_x86.conf

```diff
@@ -0,0 +1,40 @@
+# Multi-threaded, with timer support in the kernel
```

Comment:
Same here, move to `samples/helloshell/boards/qemu_x86.conf` and adapt unsupported features only.

Reply:
done